velox/expression/Expr.cpp

```diff
@@ -521,6 +522,30 @@ class ExprExceptionContext {
       sqlPath_ = e.what();
       return;
     }
+
+    if (parentExprSet_ != nullptr) {
+      std::stringstream allSql;
+      auto exprs = parentExprSet_->exprs();
+      for (int i = 0; i < exprs.size(); ++i) {
```

Comment:
I think it might be good if we standardized in saving this as a json (just use folly::dynamic). This will make it consistent with how plans/types etc are saved .

Reply:
I agree, however we would also have to change ExpressionRunner (which would be used to run this file) and expressionVerifier (that also generates this file) to be consistent. Would be good to do that refactor in a separate PR since it would a decent sized change. Filed #4959